com.unity.multiplayer.mlapi/Runtime/Core/NetworkingManager.cs

```diff
@@ -183,6 +183,13 @@ public ulong LocalClientId
         /// Gets if we are connected as a client
         /// </summary>
         public bool IsConnectedClient { get; internal set; }
+
+        /// <summary>
+        /// Gets whether or not a server or client is running.
+        /// </summary>
+        public bool IsRunning => IsServer || IsClient;
```

Comment:
Adding this for convenience to check whether the networking manager is running as a client, server or host.